BLE_GattServer_FOTAService/README.md

```diff
@@ -1,36 +1,72 @@
 # FOTA Service Mock Example
 
-**Note:** These are basic instructions intended for reviewers. A more comprehensive README is required. 
+The FOTA service, as defined in its [specification](https://github.com/ARMmbed/mbed-os-experimental-ble-services/blob/fota-service-github-ci/services/FOTA/docs/README.md) document, facilities the transfer of firmware updates over BLE.
+
+In this demo, the FOTA service is used to transfer a binary from the host PC into the flash of the target. 
+A basic _FOTA client_, implemented in Python using [bleak](https://pypi.org/project/bleak/), is used to read/write the binary stream, control and status characteristics. 
+Please refer to [Section 1](https://github.com/ARMmbed/mbed-os-experimental-ble-services/tree/fota-service-github-ci/services/FOTA/docs#fota-service-structure) of the spec to learn more about these characteristics.
+
+To verify the success of the transfer, the application computes the [SHA-256](https://en.wikipedia.org/wiki/SHA-2) of the binary and prints it to the serial port.
+The SHA-256 is also computed on the host using [sha256sum](https://man7.org/linux/man-pages/man1/sha256sum.1.html).
+The transfer is regarded as a success if the hashes are equal.
 
 ## Usage
-1. Using Mbed CLI (CMake is WIP), deploy, build and flash the FOTA target onto the board.
 
-    ```shell
-    mbed deploy && compile -t GCC_ARM -m NRF52840_DK -f 
-    ```
+### Hardware Requirements
 
-1. CLI 1 outputs a hex file to the BUILD folder, but we need a bin. Manually create it from the elf using objcopy:
+This application requires either the [DISCO_L475VG_IOT01A](https://os.mbed.com/platforms/ST-Discovery-L475E-IOT01A/) or [NRF52840_DK](https://os.mbed.com/platforms/Nordic-nRF52-DK/) platforms.
 
-    ```shell
-    arm-none-eabi-objcopy -O binary BUILD/NRF52840_DK/GCC_ARM/BLE_GattServer_FOTAService.elf BUILD/NRF52840_DK/GCC_ARM/BLE_GattServer_FOTAService.bin
-    ```
-   The file should be approx. 325 KiB in size.
+To use the Python client, the host PC must have Bluetooth capabilities, either through an inbuilt chipset or an external USB adapter.
 
-1. Create a Python virtual environment and activate it. Then, install [bleak](https://pypi.org/project/bleak/).
+### Building 
 
-   ```shell
-   cd scripts && mkdir venv && virtualenv venv && source venv/bin/activate  
-   pip install bleak==0.10.0  
-   ```
+The application can be built and flashed onto the target using Mbed CLI 2:
+
+```shell
+mbedtools deploy
+mbedtools compile -t <toolchain> -m <target> -f
```

Comment:
```suggestion
mbed-tools deploy
mbed-tools compile -t <toolchain> -m <target> -f
```

Reply:
Actually, the hyphen is not required. `mbedtools` is an alias for `mbed-tools`.